tools/packaging/project/powershell.nuspec

```diff
@@ -11,7 +11,7 @@
     <projectUrl>https://github.com/powershell/powershell</projectUrl>
     <iconUrl>https://github.com/PowerShell/PowerShell/blob/master/assets/Powershell_64.png</iconUrl>
     <description>This package contains the PowerShell Core for $runtime$.</description>
-    <copyright>Copyright © Microsoft Corporation</copyright>
+    <copyright>Copyright (c) Microsoft Corporation</copyright>
```

Comment:
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."